include/mega/posix/drivenotifyposix.h

```diff
@@ -0,0 +1,47 @@
+/**
+ * @file mega/posix/drivenotifyposix.h
+ * @brief Mega SDK various utilities and helper classes
+ *
+ * (c) 2013-2020 by Mega Limited, Auckland, New Zealand
+ *
+ * This file is part of the MEGA SDK - Client Access Engine.
+ *
+ * Applications using the MEGA API must present a valid application key
+ * and comply with the the rules set forth in the Terms of Service.
+ *
+ * The MEGA SDK is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * @copyright Simplified (2-clause) BSD License.
+ *
+ * You should have received a copy of the license along with this
+ * program.
+ */
+
+#pragma once
+
+#ifdef HAVE_DRIVE_NOTIFY
+
+// Include "mega/drivenotify.h" where needed.
+// This header cannot be used by itself.
+
+
+namespace mega {
+
+    // Posix: Platform specific definition
+    //
+    // Not implemented.
+    class DriveNotifyPosix : public IDriveNotify
+    {
+    public:
+        bool start(NotificationFunc driveConnected, NotificationFunc driveDisconnected) override;
```

Comment:
do we need parameters here?   All the implmentations of this class should be doing the same thing - queueing the details to be delivered, we don't need that to be indirect

Reply:
Yeah, I thought it'd be useful to have them separated, but if they'll never be used like that, I'll just remove one of them.